common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1142,27 +1195,13 @@ def create_xblock(
             else:
                 block_id = u'{}_{}'.format(block_type, uuid4().hex[:5])
 
+        # should only get the runtime from the bulk ops record and nest all the use of the runtim
+        # in the bulk_op, but to minimize change I'm impl'g this odd pattern here
```

Comment:
This seems like a bad call to me. What happens if the runtime from the bulk op gets used after the bulk op ends?